library/couchbase

```diff
@@ -1,16 +1,16 @@
 # maintainer: Couchbase Docker Team <docker@couchbase.com> (@couchbase)
 
-latest: git://github.com/couchbase/docker@932a91668a12f59ee081dd0e7b3ca9235ef99403 enterprise/couchbase-server/5.1.0
-enterprise: git://github.com/couchbase/docker@932a91668a12f59ee081dd0e7b3ca9235ef99403 enterprise/couchbase-server/5.1.0
-5.1.0: git://github.com/couchbase/docker@932a91668a12f59ee081dd0e7b3ca9235ef99403 enterprise/couchbase-server/5.1.0
-enterprise-5.1.0: git://github.com/couchbase/docker@932a91668a12f59ee081dd0e7b3ca9235ef99403 enterprise/couchbase-server/5.1.0
+latest: https://github.com/couchbase/docker@932a91668a12f59ee081dd0e7b3ca9235ef99403 enterprise/couchbase-server/5.1.0
+enterprise: https://github.com/couchbase/docker@932a91668a12f59ee081dd0e7b3ca9235ef99403 enterprise/couchbase-server/5.1.0
+5.1.0: https://github.com/couchbase/docker@932a91668a12f59ee081dd0e7b3ca9235ef99403 enterprise/couchbase-server/5.1.0
+enterprise-5.1.0: https://github.com/couchbase/docker@932a91668a12f59ee081dd0e7b3ca9235ef99403 enterprise/couchbase-server/5.1.0
 
-5.5.0-beta: git://github.com/couchbase/docker@932a91668a12f59ee081dd0e7b3ca9235ef99403 enterprise/couchbase-server/5.5.0-beta
+5.5.0-beta: https://github.com/couchbase/docker@932a91668a12f59ee081dd0e7b3ca9235ef99403 enterprise/couchbase-server/5.5.0-beta
 
-4.6.4: git://github.com/couchbase/docker@932a91668a12f59ee081dd0e7b3ca9235ef99403 enterprise/couchbase-server/4.6.4
-enterprise-4.6.4: git://github.com/couchbase/docker@932a91668a12f59ee081dd0e7b3ca9235ef99403 enterprise/couchbase-server/4.6.4
+4.6.5: https://github.com/couchbase/docker@932a91668a12f59ee081dd0e7b3ca9235ef99403 enterprise/couchbase-server/4.6.5
+enterprise-4.6.5: https://github.com/couchbase/docker@932a91668a12f59ee081dd0e7b3ca9235ef99403 enterprise/couchbase-server/4.6.5
```

Comment:
Most likely it is caused by this new version not existing in the old commit.

You might want to switch to the newer RFC2822 based format; it has much less repetition. You could then add multiple architectures later :wink:.

Reply:
Augh, not sure how I did that. Re-proposing. I'll look at the format change at a later date.

Reply:
Thanks for your help!